projects/plugins/jetpack/json-endpoints/jetpack/class.jetpack-json-api-cron-endpoint.php

```diff
@@ -56,6 +60,8 @@ protected function resolve_arguments() {
 	 * Check the cron lock.
 	 *
 	 * @param float $gmt_time - the time in GMT.
+	 *
+	 * @return int
```

Comment:
```suggestion
	 * @return string|int|WP_Error WP_Error if cron was locked in the `WP_CRON_LOCK_TIMEOUT` seconds before `gmt_time`, int or string otherwise.
```
I can't say that makes a whole lot of sense to me, but that's what it does.